Dart/testData/analysisServer/callHierarchy/MultiCallees/MultiCallees_verification.xml

```diff
@@ -0,0 +1,93 @@
+<node text="A.a() → B A" base="true">
+  <node text="A.a() → B A">
+    <node text="A.a() → B A">
+      <node text="A.a() → B A">
+        <node text="A.a() → B A">
+          <node text="A.a() → B A">
+            <node text="A.a() → B A">
+              <node text="A.a() → B A">
+                <node text="A.a() → B A">
+                  <node text="A.a() → B A">
+                    <node text="A.a() → B A">
+                      <Probably infinite part skipped>
+                      <Probably infinite part skipped>
+                      <Probably infinite part skipped>
```

Comment:
trick with `<Probably infinite part skipped>` doesn't work for me. I changed it to `<any/>` - it is used to stop expanding the tree in test at this point.